### PR TITLE
DOBC docker setup

### PR DESCRIPTION
**Pull request by ramereth**
This sets up docker and deploys a set of containers using a CSV file with ports
and passwords.

### CODE REVIEW

File: files/default/start-container.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+while IFS=, read -r port password ; do
+  if [ "$2" == "$port" ] ; then
+    docker run -p "${port}":22 -h dobc --rm -e DOBC_PASSWORD="${password}" \
+      --name=dobc-"${port}" -d \
+      --device-write-bps /dev/vda:10mb \
```
Comment (doublej472) [inline files/default/start-container.sh]: Reminder that you wanted to increase bps to 20mb per container

**Review by doublej472 — APPROVED** [review]
Everything looks good to me!

**Comment by ramereth** [comment]
!bump major

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v1.0.0.  Have a nice day!

**Pull request merged**